Libraries/Components/Button.windows.js

```diff
@@ -84,7 +91,32 @@ class Button extends React.Component {
      * Used to locate this view in end-to-end tests.
      */
     testID: PropTypes.string,
-  };
+    /**
+     * tabIndex:
+     * -1: Control is not keyboard focusable in any way
+     * 0: Control is keyboard focusable in the normal order
```

Comment:
>0 [](start = 7, length = 1)

` 0 (default): `

Reply:
Fixed

---
In reply to: [156829619](https://github.com/Microsoft/react-native-windows/pull/1546#discussion_r156829619) [](ancestors = 156829619)